app/gui/src/project-view/stores/graph/unconnectedEdges.ts

```diff
@@ -90,6 +90,40 @@ export function useUnconnectedEdges() {
     })
   }
 
+  const createNodeFromOutputPortButtonEdgesHovered = reactive(new Set<AstId>())
+  const createNodeFromOutputPortButtonEdgePositions = reactive(new Map<AstId, Vec2>())
```

Comment:
The naming is getting nuts… But I still don’t know anything better than `createNodeFromOutputPortButton`.

Reply:
Yeah it's unwieldy, but I don't see a good alternative